cpp/arcticdb/pipeline/frame_utils.hpp

```diff
@@ -142,12 +142,21 @@ void aggregator_set_data(
                     } else if(is_py_nan(*ptr_data)){
                         agg.set_no_string_at(col, s, nan_placeholder());
                     } else {
+                        std::variant<convert::StringEncodingError, convert::PyStringWrapper> wrapper_or_error;
```

Comment:
Do you think there will be a performance impact of constructing a variant a number of times that is linear in terms of the number of rows?

Reply:
I like to think compiler optimisations would amortize it, but can instantiate before the `for` loop to make it explicit